src/test/java/com/issoft/training/utility/Constants.java

```diff
@@ -1,11 +1,16 @@
 package com.issoft.training.utility;
 
+import org.openqa.selenium.By;
+
 public class Constants {
-    public static Integer IMPLICITLY_WAIT_TIMEOUT = 20;
-    public static Integer PAGE_LOAD_TIMEOUT = 30;
-    public static Integer POLLING = 100;
-    // create account
-    public static String INPUT_EMAIL_SCREEN = "input_email_screen.png";
-    public static String TITLE_OF_ACCOUNT_PAGE = "Login - My Store";
-    public static String HEADER_PERSONAL_INFO_PAGE_TEXT = "Your personal information";
+    public static final Integer IMPLICITLY_WAIT_TIMEOUT = 90;
+    public static final Integer PAGE_LOAD_TIMEOUT = 120;
+    public static final String INPUT_EMAIL_SCREEN = "input_email_screen.png";
+    public static final String LOGIN_EMAIL_SCREEN = "login_email_screen.png";
+    public static final By HEADER_PERSONAL_INFO_PAGE = By.xpath("//*[@id='account-creation_form']/div[1]/h3");
+    public static final String TITLE_OF_ACCOUNT_PAGE = "Login - My Store";
+    public static final By HEADER_MY_ACCOUNT_PAGE = By.xpath("//*[@id='center_column']/h1");
+    public static final String MY_WISHLIST_NAME = "My Wishlist";
+    public static final String TOTAL_PRODUCTS_IN_CART = "3";
+    public static final By MY_ACCOUNT_SIGN_OUT = By.xpath("//*[@class='logout']");
```

Comment:
All xpathes should be moved to the appropriate page classes. You shouldn't have any in constants class. It's mostly done to have everything in one place. 
Like any locator related to Login page will be stored only in appropriate LoginPage class. And it won't be any place in framework with Login page locators

Reply:
All xpathes have been moved to the appropriate page classes.